rotkehlchen/tests/utils/kraken.py

```diff
@@ -554,5 +554,10 @@ def api_query(self, method: str, req: Optional[dict] = None) -> dict:
             new_data['count'] = len(new_data['ledger'])
             response = json.dumps(new_data)
             return jsonloads_dict(response)
+        if method == 'Assets':
+            dir_path = Path(__file__).resolve().parent.parent
+            filepath = dir_path / 'data' / 'assets_only_kraken.json'
+            with open(filepath) as f:
+                return jsonloads_dict(f.read())['result']
```

Comment:
Here you are repeating logic. It would be better to abstract it to a function and just return the result of calling it